dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md

```diff
@@ -0,0 +1,116 @@
+---
+title: How to add or remove an event handler
+description: Learn how to create an event handler at design-time with the Windows Forms Designer in Visual Studio or at run-time.
+ms.date: 07/16/2021
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "Windows Forms, event handling"
+  - "event handlers [Windows Forms], creating"
+  - "run time [Windows Forms], creating event handlers at"
+  - "examples [Windows Forms], event handling"
+  - "Button control [Windows Forms], event handlers"
+  - "events [Windows Forms], connecting multiple to single event handler"
+  - "event handlers [Windows Forms], connecting events to"
+  - "menus [Windows Forms], event-handling methods for multiple menu items"
+  - "Windows Forms controls, events"
+  - "menu items [Windows Forms], multicasting event-handling methods"
+---
+
+# How to handle an event (Windows Forms .NET)
+
+Events for controls (and for forms) are generally set through the Visual Studio Visual Designer for Windows Forms. Setting an event through the Visual Designer is known as handling an event at design-time. You can also handle events dynamically in code, known as handling events at run-time.
```

Comment:
Do you need the last sentence here since you mention it again in the next paragraph?